src/main/kotlin/com/pubnub/api/PubNub.kt

```diff
@@ -1003,7 +1003,7 @@ class PubNub(val configuration: PNConfiguration) {
             sort = sort,
             includeCount = includeCount
         ),
-        withIncludes = Include(includeCustom = includeCustom)
+        withIncludes = IncludeQueryParam(includeCustom = includeCustom)
```

Comment:
I like IncludeQueryParam class name. Why not to change "withIncludes" to  "includeQueryParam" ?

Reply:
sure